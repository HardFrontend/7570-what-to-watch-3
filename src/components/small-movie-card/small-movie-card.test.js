import React from 'react';
import renderer from 'react-test-renderer';
import FilmCard from './small-movie-card.jsx';

it(`Render FilmCard`, () => {
  const tree = renderer
    .create(<FilmCard
      filmName={`Film Name`}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});


