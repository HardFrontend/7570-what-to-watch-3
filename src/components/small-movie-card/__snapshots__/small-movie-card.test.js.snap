// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render FilmCard 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseOut={[Function]}
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Bohemian rhapsody"
      height="175"
      src="img/bohemian-rhapsody.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      onClick={[Function]}
    >
      Bohemian rhapsody
    </a>
  </h3>
</article>
`;
